activesupport/lib/active_support/testing/time_helpers.rb

```diff
@@ -93,6 +99,34 @@ def travel(duration, &block)
       #   end
       #   Time.current # => Sat, 09 Nov 2013 15:34:49 EST -05:00
       def travel_to(date_or_time)
+        if block_given? && simple_stubs.stubbed?(Time, :now)
+          travel_to_nested_block_call = <<-MSG
+
+      Calling `travel_to` with a block, when we have previously already made a call to `travel_to`, can lead to confusing time stubbing.
```

Comment:
Yes true, but they are examples; users could also say `5.minutes` or `60.hours`.

I tried to clarify the previous state with:

```
Calling `travel_to` within another `travel_to` block is unsupported.
```

Doesn't that capture it good enough, while saving complexity?

I'm just trying to cut down on the exception length because I think it's overwhelming as is :smile: